bin/stt_test/utils/test_data.py

```diff
@@ -0,0 +1,45 @@
+
+from os import path, listdir, remove
+from pathlib import Path
+from typing import Dict, Literal
+import wget  # type: ignore
+import shutil
+import tarfile
+
+AUDIO_DIR = Path(path.join(path.dirname(
```

Comment:
I thought it would be better if this directory was always in one place so that there was no need to point to it when running analysis.
I also thought it would allow us to download files separately when we need them, and in this way the tool will always know this folder, which will simplify the command
If you want we can do that.

Reply:
OK let's do it that way at least for now.